decrypter.js

```diff
@@ -79,6 +79,9 @@ Splitter.prototype.flush = function () {
 
 function unpad (last) {
   var padded = last[15]
+  if (padded < 1 || padded > 16) {
+    throw new Error('invalid padding')
```

Comment:
Would you prefer the same error message as the error below? `unable to decrypt data`

Reply:
what would the error on node be ?

Reply:
"error:06065064:digital envelope routines:EVP_DecryptFinal_ex:bad decrypt"

Reply:
I felt the same way. Done.